tap-gui/plugins/runtime-resource-visibility.md

```diff
@@ -66,15 +65,82 @@ applicable for the resource type.
 To view the list of your running resources:
 
 1. Select the Component from the Catalog index page.
-1. Select the Workloads tab.
+2. Select the Runtime Resources tab.
 
-    ![Workload index table](./images/workload-visibility-workloads.png)
+![Runtime resources index table collapsed](images/runtime-resources-index.png)
 
-## Knative service details page
+### Seeing details for a specific resource
+
+The index table will show Deployments, Pods, ReplicaSets and Services that matches with the label indicated in the component's definition; you will see a master-detail structure between all the different objects using the owners and dependents relationships stablished between them; those resources without an owner will be listed in the table as independent elements.
```

Comment:
The Resources index table will show Deployments, Pods, ReplicaSets and Services that match with the label indicated in the component's definition; you will see a hierarchical structure showing the owner-dependent  relationship between the objects. Resources without an owner will be listed in the table as independent elements.